docs/spfx/viva/get-started/actions/geolocation/GeolocationTutorial.md

```diff
@@ -0,0 +1,262 @@
+---
+title: Hands on tutorial for creating an Adaptive Card Extension Card with geolocation action
+description: Step by step guide on creating Adaptive Card Extension for Geolocation action.
+ms.date: 03/04/2022
+ms.prod: sharepoint
+ms.localizationpriority: high
+---
+# Hands on tutorial for creating an Adaptive Card Extension with geolocation action
+> [!NOTE]
+> This tutorial assumes that you have installed the SPFx v1.14<br/>
+> For more information on installing the SPFx v1.14, see [SharePoint Framework v1.14 release notes](../../../../release-1.14.md).
+
+## Scaffold an Adaptive Card Extension project
```

Comment:
I'm not sure if we need the scaffolding section here or just reference "get started" tutorial.
@VesaJuvonen @johnguy0 - thoughts?

Reply:
The only reason why I added it was because in the instructions I suggest devs to name their ACE: "GeoLocation" and then everywhere in the code I ask them to go to a file path, which contains this name.

If I remove it, then those devs who are following it from the beginning might find it a bit confusing/annoying.

Reply:
I was asked to review this PR and merge it if everything was good. I made lots of fixes to the markdown, but this requested change is still present. Can't tell if the discussion is resolved. 

@AJIXuMuK / @SidGulatiMsft, can you either make the necessary change, or mark it as resolved so I can merge this PR?

Reply:
@VesaJuvonen , @johnguy0, could you please help confirm if we could mark this comment as resolved?

Reply:
Let's keep this in to align on the tutorial model which we in general use within the SPFx docs. Section is not long and provides the needed context on getting started - So let's keep it there for now.

Reply:
@andrewconnell, thanks for making check-in ready changes. Please go ahead and merge the PR.

Reply:
Changing doc titles to match conventions in the rest of the site. Doesn't make sense to say "Documentation for... " when that's what the whole site is about :)

Will also get it added to the TOC